build.gradle

```diff
@@ -1,14 +1,11 @@
-group = 'com.github.aphoh'
-version = '0.0.1-SNAPSHOT'
-
-buildscript {
-    repositories {
-    }
-    dependencies {
-    }
-}
-
 allprojects {
+    apply plugin: 'java'
+    group = 'com.pokegoapi'
+    version = '0.3-1'
```

Comment:
This does not match the version number you provided in https://github.com/Grover-c13/PokeGOAPI-Java/pull/453. There needs to be consistency in versioning.